lib/pages/assignment/check_assignment.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'assignment_animation.dart'; // Ensure this is the correct path for AssignmentAnimationManager
+import 'widgets/check_assingment_appbar.dart';
+import 'widgets/student_list.dart';
+import 'widgets/close_button_widget.dart';
+
+class CheckAssignmentPage extends StatefulWidget {
+  const CheckAssignmentPage({super.key});
+
+  @override
+  _CheckAssignmentPageState createState() => _CheckAssignmentPageState();
+}
+
+class _CheckAssignmentPageState extends State<CheckAssignmentPage>
+    with SingleTickerProviderStateMixin {
+  late AssignmentAnimationManager animationManager;
+  final double appBarHeight = 100.0; // Default AppBar height
+
+  @override
+  void initState() {
+    super.initState();
+    animationManager = AssignmentAnimationManager(vsync: this);
+    _startAnimation();
+  }
+
+  void _startAnimation() {
+    animationManager.startAnimation();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        children: [
+          Container(
+            color: const Color(0xFF5278C1), // Background remains the same
+          ),
+          Positioned(
+            top: appBarHeight, // Start below the AppBar
+            left: 0,
+            right: 0,
+            bottom: 0, // Extend to the bottom
+            child: AnimatedBuilder(
+              animation: animationManager.controller,
+              builder: (context, child) {
+                return Container(
+                  decoration: const BoxDecoration(
+                    color: Colors.white,
+                    borderRadius: BorderRadius.only(
+                      topLeft: Radius.circular(30.0),
+                      topRight: Radius.circular(30.0),
+                    ),
+                  ),
+                  child: Column(
+                    children: [
+                      Expanded(
+                        child: StudentList(),
+                      ),
+                      const CloseButtonWidget(),
+                    ],
+                  ),
+                );
+              },
+            ),
+          ),
```

Comment:
consider to remove animated builder here since there is no active animation yet. return it when there is a need to implement some animation